package.json

```diff
@@ -6,16 +6,19 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "node dist/main.js -p 8080 --log-level=debug",
-    "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --use-cache",
+    "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --port=8040",
     "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js' --require @babel/register --require test_setup.js",
     "deploy": "./deploy.sh",
     "deploy:docker": "DOCKER=1 ./deploy.sh",
     "set_default": "./set_default.sh",
     "lint": "flow && eslint --config .eslintrc ./*.js src",
     "prettyquick": "pretty-quick",
-    "build:noclean": "babel src --out-dir dist --source-maps --ignore src/testdata",
+    "build:noclean": "babel src --out-dir dist --source-maps --ignore src/testdata --config-file ./babel.config.js",
     "build": "npm run clean && npm run build:noclean",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "check:file": "bash -c 'git diff \"${@:1}\" --shortstat -- $0'",
+    "error:file-change": "bash -c 'if [ \"$(npm run --silent check:file -- ${@:0})\" != \"\" ]; then echo \"🛑  $0 has changed - commit the changes\" && exit 1 ; else echo \"✅  $0 is up-to-date\" ; fi'",
+    "update:yarn.lock": "if [ \"$(npm run --silent check:file -- package-lock.json --cached)\" != \"\" ] ; then (rm yarn.lock 2> /dev/null || true) && yarn import ; fi && npm run --silent error:file-change -- yarn.lock"
```

Comment:
I'm confused as to what this is doing - why are we looking at `package-lock.json`? Shouldn't that file be removed?

Reply:
No, this project has not moved to yarn. That's a bigger change. It's still using NPM. We're just adding a yarn lock file so that it works if installed via yarn, like webapp does.

Reply:
Ahh, ok! That's pretty confusing, given that we're wanting to generate a `yarn.lock` file and keep it up-to-date!

Reply:
Yup. Sadly, in a world with multiple package managers, we need to worry about this when we're not shrinkwrapping and publishing.

I see this as temporary, given the other work we're doing